lib/features/thread/data/repository/thread_repository_impl.dart

```diff
@@ -249,9 +248,7 @@ class ThreadRepositoryImpl extends ThreadRepository {
 
   @override
   Stream<EmailsResponse> loadMoreEmails(GetEmailRequest emailRequest) async* {
-    bench.start('loadMoreEmails in computed');
-    final response = await compute(_getAllEmailsWithoutLastEmailId, emailRequest);
-    bench.end('loadMoreEmails in computed');
+    final response = await _getAllEmailsWithoutLastEmailId(emailRequest);
```

Comment:
Can you share the root cause? Any relate to `httpClient` in `isolate` environment?